### PR TITLE
new case about vm destroy with nwfilter

### PR DESCRIPTION
**Pull request by juteman**
while vm destroy with nwfilter, libvirtd.log
will get error before. and get check
Signed-off-by: qiulzhan <qiulzhan@redhat.com>

### CODE REVIEW

**Comment by juteman** [comment]
```
JOB ID     : 17246a8d5506893efed8777dd2a156697545e337
JOB LOG    : /root/avocado/job-results/job-2018-08-28T17.35-17246a8/job.log
 (1/1) type_specific.io-github-autotest-libvirt.vm_destroy_with_nwfilter.check_libvirtd_log: PASS (11.88 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 22.84 s

```

File: libvirt/tests/cfg/nwfilter/vm_destroy_with_nwfilter.cfg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- vm_destroy_with_nwfilter:
+    type = vm_destroy_with_nwfilter
+    start_vm = "yes"
+    status_error = "no"
+    kill_vm = "yes"
```
Comment (chloerh) [inline libvirt/tests/cfg/nwfilter/vm_destroy_with_nwfilter.cfg]: This line might not be necessary. vm will be destroyed by default.

**Comment by chloerh** [comment]
The rest is ok to me.

**Comment by kylazhang** [comment]
Merge the PR as he left and the update will be given later if needed

**Pull request merged**